src/e2e/java/teammates/e2e/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -911,4 +963,109 @@ private void clickSaveNewQuestionButton() {
     private WebElement getRecommendedTextLengthField(int questionNum) {
         return getQuestionForm(questionNum).findElement(By.id("recommended-length"));
     }
+
+    private WebElement getWeightCheckbox(int questionNum) {
+        return getQuestionForm(questionNum).findElement(By.id("weights-checkbox"));
+    }
+
+    private String getDoubleString(Double value) {
+        return value % 1 == 0 ? Integer.toString(value.intValue()) : Double.toString(value);
+    }
+
+    private WebElement getRubricRow(int questionNum, int rowNumber) {
+        return getQuestionForm(questionNum).findElements(By.cssSelector("tm-rubric-question-edit-details-form tr"))
+                .get(rowNumber - 1);
+    }
+
+    private List<WebElement> getRubricChoiceInputs(int questionNum) {
+        return getRubricRow(questionNum, 1).findElements(By.tagName("input"));
+    }
+
+    private List<WebElement> getRubricTextareas(int questionNum, int rowNum) {
+        return getRubricRow(questionNum, rowNum).findElements(By.tagName("textarea"));
+    }
+
+    private List<WebElement> getRubricWeights(int questionNum, int rowNum) {
+        return getRubricRow(questionNum, rowNum).findElements(By.tagName("input"));
+    }
+
+    private WebElement getRubricDeleteSubQnBtn(int questionNum, int rowNum) {
+        return getRubricRow(questionNum, rowNum).findElement(By.id("btn-delete-subquestion"));
+    }
+
+    private WebElement getRubricDeleteChoiceBtn(int questionNum, int colNum) {
+        return getRubricRow(questionNum, getNumRubricRows(questionNum)).findElements(By.id("btn-delete-choice")).get(colNum);
+    }
+
+    private int getNumRubricRows(int questionNum) {
+        return getQuestionForm(questionNum).findElements(By.cssSelector("tm-rubric-question-edit-details-form tr")).size();
```

Comment:
Is there a better selector here? While this works, having to rely on the Angular component name is not preferred.